### PR TITLE
[SMALLFIX] Use Whitebox instead of exposing setTTLIntervalMs method

### PR DESCRIPTION
**Pull request by aaudiber**
Followup to https://github.com/amplab/tachyon/pull/2188

### CODE REVIEW

File: servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java [hunk 1]
```diff
@@ -62,6 +64,7 @@
   private static final TachyonURI ROOT_FILE_URI = new TachyonURI("/file");
   private static final TachyonURI TEST_URI = new TachyonURI("/test");
   private static CreateOptions sNestedFileOptions;
+  private static long sOldTtlIntervalMs;
```
Comment (jsimsa) [inline servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java]: nit: should we either user Ttl or TTL consistently?

**Comment by jsimsa** [comment]
@aaudiber thanks, just one minor comment

File: servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java (hunk 1 above)
Comment (aaudiber) [inline servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java]: Our style guide says we should Ttl. I'll create a ticket + followup followup PR for it.

Comment (aaudiber) [inline servers/src/test/java/tachyon/master/file/FileSystemMasterTest.java]: https://tachyon.atlassian.net/browse/TACHYON-1387

**Comment by jsimsa** [comment]
LGTM

File: servers/pom.xml [hunk 2]
```diff
@@ -148,6 +148,18 @@
         <artifactId>maven-jspc-plugin</artifactId>
         <version>2.0.8</version>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```
Comment (yupeng9) [inline servers/pom.xml]: Why is this needed?

Comment (aaudiber) [inline servers/pom.xml]: This causes maven to generate a test jar which other modules can depend on.

Comment (yupeng9) [inline servers/pom.xml]: oh, I see. `TTLBucketPrivateAccess` is defined in test package. Thanks.

**Comment by yupeng9** [comment]
LGTM
